ssh/tailssh/tailssh_test.go

```diff
@@ -265,6 +265,8 @@ func TestSSH(t *testing.T) {
 
 	execSSH := func(args ...string) *exec.Cmd {
 		cmd := exec.Command("ssh",
+			"-F",
+			"none",
```

Comment:
/dev/null?

Reply:
No, because that means I have to check if `NUL:` works on with ssh on Windows, and I was being lazy.

`none` is an ssh magic value.

Reply:
Well, turns out `NUL:` works too, but like I said, `none` is a well-known value to ssh meaning no config:

```
C:\Users\raggi>ssh -F foo ...
Can't open user config file foo: No such file or directory

C:\Users\raggi>ssh -F none rag.pub
Warning: the ECDSA host key for '...' differs from the key for the IP address '...'
```